openid-4-verifiable-presentations-1_0.md

```diff
@@ -2350,7 +2352,7 @@ nonce = tstr
 The `OpenID4VPDCAPIHandover` structure has the following elements:
 
 * The first element MUST be the string `OpenID4VPDCAPIHandover`. This serves as a unique identifier for the handover structure to prevent misinterpretation or confusion.
-* The second element MUST be a bytestring which contains the sha-256 hash of the bytes of `OpenID4VPDCAPIHandoverInfo` when encoded as CBOR.
+* The second element MUST be a byte string which contains the sha-256 hash of the bytes of `OpenID4VPDCAPIHandoverInfo` when encoded as CBOR.
```

Comment:
Not sure about this one - but all CBOR related rfcs always use "Byte String", whereas a lot of other places use "Bytestring". I thought it would make sense to keep things similar to the RFCs here.